app/assets/javascripts/price_policy.js.coffee

```diff
@@ -1,65 +1,57 @@
 $(document).ready ->
-  $interval = $("#interval")
 
-  $interval.change ->
-    int_value = $(this).val()
-    interval = "#{int_value} minute"
-    interval += "s" if int_value > 1
-    $("span.interval_replace").each -> $(this).html(interval)
+  isFiniteAndPositive = (number) -> isFinite(number) && number > 0
 
-  $interval.trigger('change')
+  # In addition to disabling the field, also hide its value. But still store it
+  # so we can display it again if it gets renabled
```

Comment:
Do you know why this is necessary? It seems to me like it would be enough (and simpler) to just disable the field…

Reply:
I'm guessing our reasoning was that if you just set `disabled` on the the field, you still see the value that was there before it got disabled, though the field is grayed out. To me, it's confusing to see a value when it's not going to be used. This is especially true in with the new toggle for full cancellation.

Reply:
Gotcha, thanks!